lib/endpoint/tags/table.js

```diff
@@ -126,6 +126,47 @@ function rowToParameter (i, el) {
     params.type = 'objects[]'
   }
 
+  if (params.name === 'content') {
+    if (/reaction type/.test(params.description) === true) {
+      params.enum = ['+1',
+        '-1',
+        'laugh',
+        'confused',
+        'heart',
+        'hooray']
+    }
+  }
+
+  if (params.name === 'type') {
+    if (/Normally this is a `commit`/.test(params.description) === true) {
```

Comment:
I think this is such a special case, we can as well check for the entire sentence:

```
/Normally this is a `commit` but it can also be a `tree` or a `blob/
```